src/components/button-group/button-group.ts

```diff
@@ -109,7 +109,7 @@ export default class IgcButtonGroupComponent extends EventEmitterMixin<
   }
 
   public set selectedItems(values: string[]) {
-    if (values && values.length) {
+    if (values) {
       this._selectedItems = new Set(values);
       this.setSelection(this._selectedItems);
     }
```

Comment:
```suggestion
    this._selectedItems = new Set(Array.isArray(values) ? values : []);
    this.setSelection(this._selectedItems);
```
This would also work when assigning falsy values, i.e. `null/undefined`.
You can add a test case for that as well.